### PR TITLE
add model location for demo

### PR DESCRIPTION
**Pull request by weiguoz**
Follow [PR:Save model on the local filesystem](https://github.com/sql-machine-learning/sqlflow/pull/488)

### CODE REVIEW

**Review by Yancey0623 — APPROVED** [review]
LGTM

File: cmd/demo/demo.go [hunk 1]
```diff
@@ -83,6 +83,7 @@ func render(rsp interface{}, table *tablewriter.Table) bool {
 
 func main() {
 	ds := flag.String("datasource", "", "database connect string")
+	modelDir := flag.String("model_dir", "", "model would be saved on the local dir, otherwise upload to the table.")
```
Comment (wangkuiyi) [inline cmd/demo/demo.go:86]: "save the model to SQL engine as a human-unreadable table if this option is empty"

File: cmd/demo/demo.go [hunk 2]
```diff
@@ -101,7 +108,7 @@ func main() {
 		isTable, tableRendered := false, false
 		table := tablewriter.NewWriter(os.Stdout)
 
-		stream := sql.Run(slct, db, "")
+		stream := sql.Run(slct, db, *modelDir)
```
Comment (wangkuiyi) [inline cmd/demo/demo.go:111]: Is it a better idea to create the modelDir in `sql.Run` than here in `main`? I vaguely feel that it is easier to maintain the code if we allocate the resource right before we use it.

**Review by wangkuiyi — APPROVED** [review]

File: cmd/demo/demo.go (hunk 2 above)
Comment (weiguoz) [reply cmd/demo/demo.go:111]: Good idea.
SQLFlow needs a model directory only a SQL is extended.

**Pull request merged**